sbc_compassion/models/contracts.py

```diff
@@ -39,6 +39,10 @@ class Contracts(models.Model):
         'Days since sponsor wrote',
         compute='_compute_last_letter'
     )
+    incite_to_write = fields.Boolean(
+        'Increase visibility of writing content',
```

Comment:
Add a hint on the purpose on the field (beware of line too long because of PEP8)
```suggestion
        help='True when child will have birthday in less than 3 months and sponsor has not written in this period.',
```